src/templates/dashboard.html

```
@@ -116,7 +116,7 @@ <h3><a href='scheduler.php'><i class='far fa-calendar-alt mr-1'></i>{{ 'Schedule
     </div>
     <hr>
     <div class='list-group mt-1' id='scheduledBookings'>
-      {% for booking in bookingsArr %}
+      {% for booking in bookingsArr|sort((a, b) => a.start <=> b.start) %}
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)